gen/definitions/banner.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: Banner
+path: Cisco-IOS-XR-um-banner-cfg:/banners/banner[banner-type=%s]
+doc_category: System
+attributes:
+  - yang_name: banner-type
+    example: prompt-timeout
```

Comment:
Could we maybe change the example type to `login` as this would be a typical banner one would set.

Reply:
change banner type to login